bzt/modules/selenium.py

```diff
@@ -502,3 +516,152 @@ def __init__(self, tool_path, download_link):
 
     def install(self):
         raise NotImplementedError()
+
+
+class NoseTest(object):
+    IMPORTS = """import unittest
+import re
+from time import sleep
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+from selenium.common.exceptions import NoAlertPresentException"""
+
+    def __init__(self):
+        self.root = etree.Element("NoseTest")
+        self.tree = etree.ElementTree(self.root)
+
+    def add_imports(self):
+        imports = etree.Element("imports")
+        imports.text = NoseTest.IMPORTS
+        return imports
+
+    def gen_class_definition(self, class_name, inherits_from, indent="0"):
+        def_tmpl = "class {class_name}({inherits_from}):"
+        class_def_element = etree.Element("class_definition", indent=indent)
+        class_def_element.text = def_tmpl.format(class_name=class_name, inherits_from="".join(inherits_from))
+        return class_def_element
+
+    def gen_method_definition(self, method_name, params, indent="4"):
+        def_tmpl = "def {method_name}({params}):"
+        method_def_element = etree.Element("method_definition", indent=indent)
+        method_def_element.text = def_tmpl.format(method_name=method_name, params=",".join(params))
+        return method_def_element
+
+    def gen_method_statement(self, statement, indent="8"):
+        statement_elem = etree.Element("statement", indent=indent)
+        statement_elem.text = statement
+        return statement_elem
+
+
+class SeleniumScriptBuilder(NoseTest):
+    def __init__(self, scenario, parent_logger):
+        super(SeleniumScriptBuilder, self).__init__()
+        self.log = parent_logger.getChild(self.__class__.__name__)
+        self.scenario = scenario
+
+    def gen_test_case(self):
+        self.log.debug("Generating Test Case test method")
+        imports = self.add_imports()
+        self.root.append(imports)
+        test_class = self.gen_class_definition("TestRequests", ["unittest.TestCase"])
+        self.root.append(test_class)
+        test_class.append(self.gen_setup_method())
+        requests = self.scenario.get("requests")
```

Comment:
There is a method get_requests in Executor class. It has all things like think-time in it. Why not use it?